en/docs/references/extend/rebranding/customizable-login-portal.md

```diff
@@ -0,0 +1,134 @@
+    
+    - If the value of `isPolicyPage` is `true` or any non-empty string then the content inside the block will be executed.
+    - For all other values including null and undefined, the content of the block will not be executed.
+
+- **Not Condition Block**
+
+    ![Not Condition Block]({{base_path}}/assets/img/references/not-condition-block.png)
+    
+    This is similar to an `IF` block with a NOT condition. 
+    
+    - If the value of `isPolicyPage` is `false`, an empty string, null or undefined, the content inside the block will be executed. 
+    - For all other values, the content of the block will not be executed.
+
+- **Component Syntax**
+    
+    ![Component Syntax]({{base_path}}/assets/img/references/component-syntax.png)
+
+    This syntax will be used to indicate the position of the general component in the custom layout code. The component syntaxes will be replaced with actual content at runtime.
+
+- **Data Syntax**
+    
+    ![Data Syntax]({{base_path}}/assets/img/references/data-syntax.png)
+
+    This syntax can be used to add data to the layout code. The value stored in the `containerSize` variable will be converted to a string and placed in the corresponding location at runtime. All data syntax values will be sanitized before adding to the layout code.
+
+!!! info
+    All these variables will be provided by `JSP` pages. To get more data from JSP pages, add the data into the `layoutData` Map object from each JSP page. This `layoutData` map object can be found on each JSP page.
+
+
+
+## Add a custom layout for the authentication/recovery pages
+
+To customize these pages, follow the steps below.
+
+!!! note
+    On this page, we will be referring to `<IS_HOME>/repository/deployment/server/webapps/` as `webapps` folder.
+    
+### Step 1: Create the extensions folder
+
+Navigate to `webapps/authenticationendpoint` and create a new folder named `extensions`.
+
+!!! note
+    Repeat the above step for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+
+### Step 2: Create the basic custom layout resources
+
+To create the basic custom layout resources:
+
+1. On the `authenticationendpoint` folder, navigate to the created `extensions` folder and create `layouts` folder and `customs` folders recursively.
+    
+    ```
+    └── extensions
+        └── layouts
+            └── custom
+    ```
+
+3. Navigate to `webapps/authenticationendpoint/includes/layouts/default` folder, copy the `body.html` file and add it to the created `custom` folder.
+
+4. Create the following files and folder inside the `custom` folder:
+    - `styles.css` file
+    - `script.js` file
+    - `assets` folder
+
+!!! note
+    Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+
+### Step 3: Add a custom layout design
+
+To add a custom layout to the login page
+
+1. Navigate to `webapps/authenticationendpoint/extensions/layouts/custom/`
```

Comment:
incomplete

Reply:
`Navigate to `webapps/authenticationendpoint/extensions/layouts/custom/` and copy the content as instructed below:`

Is this okay? Sub-points 1,2,3,4 should be done while the user is in this folder